requirements.txt

```diff
@@ -13,4 +13,8 @@ https://github.com/MapStory/django-maploom/archive/mapstory.zip
 django-haystack==2.4.0
 elasticsearch==1.6.0
 django-slack==4.1.0
-django-oauth2-provider
\ No newline at end of file
+
+#specific dependency versions from mapstory v1 to support oauth and social authnz
+django-oauth2-provider==0.2.6.1
+django-social-auth==0.7.22
```

Comment:
THIS LIBRARY IS DEPRECATED IN FAVOR OF python-social-auth, however, we will use it in order to get the feature up and running in the near-term. We can revisit migrating to python-social-auth at a later date. My initial attempt to use python-social-auth with mapstory posed a few issues with conflicting packages required as dependencies.